properties.schema

```diff
@@ -107,14 +107,15 @@
       "validators": [],
       "help": "If set to 'true', the items in the open popup will be allowed to cycle through continiously"
     },
-    "_useGraphicsAsPins": {
```

Comment:
It's only been kept in for backwards compatibility. It's functionality can be achieved by `_pinStyle: tiles` .